web/src/components/new/TagCreateDialog.vue

```diff
@@ -7,6 +7,20 @@
         </v-card-title>
         <v-card-text>
           <v-text-field v-model="tagName" label="Name" autofocus></v-text-field>
+          <v-text-field v-model="tagColor" label="Color" hide-details>
```

Comment:
there is an issue with this, when trying to use this text field, it likely isn't able to parse the color and resets it or so

Reply:
Yeah, the text-color widget enforces the format after it was opened once. We could decouple it from the text field and manually link them manually once the text input contains a valid #XXXXXX value?

Reply:
implemented it based on if the picker is open (on open, the textbox value is set as value for the picker, while open, the value of the picker is copied into the textbox)